architecture/secure_storage.rst

```diff
@@ -334,6 +334,53 @@ a ``TEE_RPC_RPMB_CMD`` message, thanks to the ``thread_rpc_cmd()`` function.
 Most RPMB requests and responses use the data frame format defined by the JEDEC
 eMMC specification. HMAC authentication is implemented here also.
 
+Security considerations
+=======================
+The RPMB partition in eMMC can not be accessed until a key has been programmed
+on the device: this is a one time action for the lifetime of the device. Once
+the key has been written on the eMMC controller, the controller uses it to
+authenticate requests.
+
+In order to be able to program this key, OP-TEE must be configured with
+``CFG_RPMB_WRITE_KEY=y``.
```

Comment:
I would not say "OP-TEE must be configured with ``CFG_RPMB_WRITE_KEY=y``".

This is a way to program the RPMB key but there are others. For example one can use U-Boot shell command `mmc rpmb key ...`, assuming one knows the key to be programmed. There are of course other alternate ways.

Reply:
sure that is true but if the user wants to use OP-TEE to program the key this is the only way. it can be programmed by OP-TEE or Linux too...I'll try to make it mor explicit